lib/cmd/fhc/target.js

```diff
@@ -56,6 +56,7 @@ function target (argv, cb) {
     function (cb){
       //First, checking if the target platform version is greater than a minimum value.
       version.checkTargetVersion([tar], cb);
+      fhc.config.set("fhversion", 3);
```

Comment:
@wei-lee I understood that the goal of https://issues.jboss.org/browse/RHMAP-7690 is to setup it when run the command target <domain>. It should be the default value = 3. The rest of the code related it still the same. If the user make fhc login and they are using the platform 2 this value will be replaced.